homeassistant/components/flume/strings.json

```diff
@@ -8,7 +8,7 @@
     "step": {
       "user": {
         "description": "In order to access the Flume Personal API, you will need to request a 'Client ID' and 'Client Secret' at https://portal.flumetech.com/settings#token",
-        "title": "Connect to your Flume Account",
+        "title": "Connect to your Flume account",
```

Comment:
Even if they used a capital A, I think having an account is usually not that strong of a brand to make special (there probably are exceptions, but this is not the case for the majority